analyzers/src/SonarAnalyzer.CSharp/SymbolicExecution/Roslyn/SecureRandomSeedsShouldNotBePredictable.cs

```diff
@@ -91,11 +91,12 @@ private static ProgramState ProcessSecureRandomCreation(ProgramState state, IObj
 
         CryptographicSeedConstraint GeneratorConstraint() =>
             objectCreation.ArgumentValue("generator") is { } generator
-                ? state[generator].Constraint<CryptographicSeedConstraint>()
+                ? state[generator]?.Constraint<CryptographicSeedConstraint>()
                 : null;
 
         bool HasSmallAutoseed() =>
-            objectCreation.ArgumentValue("autoSeedLengthInBytes") is null or { ConstantValue.HasValue: true, ConstantValue.Value: < 16 };
+            objectCreation.ArgumentValue("autoSeedLengthInBytes") is not { } value
+            || (state[value]?.Constraint<NumberConstraint>() is { } numberConstraint && numberConstraint.Min < 16);
```

Comment:
Can you explain this to me?
Are you being conservative about raising ONLY when we know for sure that it's smaller than 16, instead of "it _might_ be smaller than 16"?

Reply:
Yes. You treat unknown seeds as big enough, right? So a seed with a number constraint like `[0, null]` should not trigger the rule either.

Reply:
Good point, thanks!